blivetgui/blivet_utils.py

```
@@ -1612,8 +1614,12 @@ def create_disk_label(self, blivet_device, label_type):
                                               label_type=label_type)
         actions.append(blivet.deviceaction.ActionCreateFormat(blivet_device, new_label))
 
-        for ac in actions:
-            self.storage.devicetree.actions.add(ac)
+        try:
+            for ac in actions:
+                self.storage.devicetree.actions.add(ac)
+        except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)